Floofbot/Modules/NicknameAlert.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Discord.Commands;
+using Floofbot.Services.Repository;
+using Floofbot.Services.Repository.Models;
+using Serilog;
+using System;
+using System.Threading.Tasks;
+
+namespace Floofbot.Modules
+{
+    public class NicknameAlert : ModuleBase<SocketCommandContext>
+    {
+        private FloofDataContext _floofDB;
+
+        public NicknameAlert(FloofDataContext floofDB)
+        {
+            _floofDB = floofDB;
+        }
+
+        protected void CheckServer(ulong server)
+        {
+            // checks if server exists in database and adds if not
+            var serverConfig = _floofDB.LogConfigs.Find(server);
+            if (serverConfig == null)
+            {
+                _floofDB.Add(new NicknameAlertConfig
+                {
+                    ServerId = server,
+                    Channel = 0,
+                    IsOn = false
+                }) ;
+                _floofDB.SaveChanges();
+            }
+        }
+
+        [Command("setchannel")] // update into a group
```

Comment:
Wait a minute, are these commands not in a commandgroup?
It should be something like .nicknamealert setchannel right?

Reply:
fixed, sorry